src/server.c

```diff
@@ -3312,6 +3346,38 @@ int populateCommandTableParseFlags(struct redisCommand *c, char *strflags) {
     if (!(c->flags & CMD_CATEGORY_FAST)) c->flags |= CMD_CATEGORY_SLOW;
 
     sdsfreesplitres(argv,argc);
+
+    /* Now handle the key arguments spec flags */
+
+    /* Redis commands don't need more args than STATIC_KEYS_SPECS_NUM (Number of keys
+     * specs can be greater than STATIC_KEYS_SPECS_NUM only for module commands) */
+    c->keys_specs = c->keys_specs_static;
+    c->keys_specs_max = STATIC_KEYS_SPECS_NUM;
+
+    for (int i = 0; i < STATIC_KEYS_SPECS_NUM; i++) {
+        if (c->keys_specs[i].type == KSPEC_INVALID)
```

Comment:
Like the moduleCommand command, which is not initialized with keys_specs_static, could it cause unforeseen problems?

Reply:
not sure i understand, moduleCommand is also handles in this function

Reply:
Sorry, I feel that my understanding is wrong.